knowhere/index/vector_index/helpers/IndexParameter.h

```diff
@@ -34,6 +34,7 @@ constexpr const char* TOPK = "k";
 constexpr const char* RADIUS = "radius";
 constexpr const char* INPUT_IDS = "input_ids";
 constexpr const char* DEVICE_ID = "gpu_id";
+constexpr const char* DATA_PATH = "data_path";
```

Comment:
I think "DATA_PATH" should put into IndexParam

Reply:
Since I use it in `DatasetPtr`, and noticed all `DatasetPtr` keys is in the `meta` namespace. I kept this convention in case there is any hidden meanings. Plz let me know if no, and I will put it in to the `indexparam` namespace.

Reply:
The data in Dataset is vector related, data in Config is index related.
"DATA_PATH" is index related, not vector related, so I think it should be in IndexParam and passed in via Config